HoneyBadgers.RestApi/Models/BaseEntity.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace HoneyBadgers.RestApi.Models
+{
+    public class BaseEntity
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
Tak w skrócie, co to robi? coś z bazą danych ale pierwszy raz widzę:O

Reply:
Jeżeli dobrze ogarniam to taki bazowy szablon dla entity, Coś jak klasa `Object` dla pozostałych klas.

Reply:
To jest coś o czym dawno dawno temu wspominał Patryk Sz. Tworzymy taki base a następnie repository ktore będzie korzystał z klasy dziedziczące po tym dzięki temu mamy jeden repository dla wszystkich